client/components/CheckIn.js

```diff
@@ -63,4 +74,12 @@ const mapState = state => {
     }
 }
 
-export default connect(mapState)(Checkout)
\ No newline at end of file
+const mapDispatch = dispatch => {
+    return {
+        fetchNearbyPlacesNextExperience: function (lat, long) {
+            dispatch(fetchNearbyPlacesNextExperience(lat, long))
```

Comment:
nit: haven't written this down anywhere, but mixing in ES6 and ES5 here my suggestion would be to: 

fetchNearbyPlacesNextExperience: (lat, long) => { dispatch(fetchNearbyPlacesNextExperience) } 

also naming convention is kinda long -- all things that are not necessary but could potential improvements